config.rb.sample

```diff
@@ -1,4 +1,7 @@
-$new_discovery_url='https://discovery.etcd.io/new'
+# Size of the CoreOS cluster created by Vagrant
+$num_instances=1
```

Comment:
@termie Please remove the `$num_instances` and accompanying comment on lines 35-36. Additionally, could you please add a comment above `$new_discovery_url` explaining the importance of defining `$num_instances` beforehand for the templating to work?